src/renderer/managers/ignition.ts

```diff
@@ -75,3 +75,19 @@ export async function ignite(): Promise<void> {
   await componentsReady();
   await start();
 }
+
+export async function startSplash(): Promise<void> {
+  log("Ignition", "Start", void 0, "Igniting Replugged Splash Screen...");
+  const startTime = performance.now();
+
+  await themes.loadMissing().then(themes.loadAll);
+  // Quick CSS needs to be called after themes are loaded so that it will override the theme's CSS
+  quickCSS.load();
```

Comment:
I don't think we want quick css here

Reply:
You mean no quickcss should be executed in the splash screen?